zetaclient/chains/evm/observer/v2_tracker.go

```diff
@@ -0,0 +1,88 @@
+package observer
```

Comment:
can we rename as `v2_inbound_tracker.go` to be explicit and consistent with solana.